Source/quests.cpp

```diff
@@ -144,7 +144,7 @@ void InitQuests()
 		quests[z]._qmsg = questlist[z]._qdmsg;
 	}
 
-	if (gbMaxPlayers == 1) {
+	if (gbMaxPlayers == 1, allquests == 0) {

```

Comment:
what the fuck

Reply:
What is inside of that if is the code that removes one of the quests from each set of quests for the randomization. So if the switch is used it changes allquests to 1 and skips that whole section and all the quests are used.

Reply:
yes, but who codes like this? xD use proper && / || instead of this voodo, + why would you make a 0/1 variable int?

Reply:
Sorry, not much of coder, more just hacking around with it. 
So it should (gbMaxPlayers == 1 || allquests == 0)?
and would it be a bool then?

Reply:
I think wat qndel is suggesting is to make allquests a bool instead of it, and then check if it's false with `!allquests` instead of `allquests == 0`

Reply:
yes + using TRUE/FALSE instead of 0/1

Reply:
ok implemented those changes